api/app/graphql/types/input/node_exclusion_input_type.rb

```diff
@@ -0,0 +1,10 @@
+module Types
+  module Input
+    class NodeExclusionInputType < Types::BaseInputObject
+      argument :excluding_node_id, ID, required: false
+      argument :excluded_node_id, ID, required: false
```

Comment:
All required a part destroy

Reply:
Can't do that, s'il veut juste supprimer, il me passera l'ID et le _destroy